scripts/release/tests/regex-tests.test.js

```diff
@@ -10,23 +11,61 @@ dependencies: [
     .package(name: "GRDB", url: "https://github.com/duckduckgo/GRDB.swift.git", .exact("1.2.0")),
 ]`
-        const examplePackageResolved = `
+        const examplePackageResolvedForBSK = `
 {
-    "package": "Autofill",
-    "repositoryURL": "https://github.com/duckduckgo/duckduckgo-autofill.git",
-    "state": {
-        "branch": null,
-        "revision": "d29c5abc7f473b69f3d81d8a15e137ed3aab029d",
-        "version": "5.0.1"
+    "identity" : "duckduckgo-autofill",
+    "kind" : "remoteSourceControl",
+    "location" : "https://github.com/duckduckgo/duckduckgo-autofill.git",
+    "state" : {
+        "revision" : "4aee97d550112ba6551e61ea8019fb1f1a2d3af7",
+        "version" : "6.4.3"
     }
 }`
 
+        const examplePackageResolvedForMacOS = `
+{
+        "identity" : "browserserviceskit",
+        "kind" : "remoteSourceControl",
+        "location" : "https://github.com/duckduckgo/BrowserServicesKit",
+        "state" : {
+            "revision" : "fc9ada07283575e3106e6fc5c670a96c34a86f55",
+            "version" : "54.1.0"
+        }
+    },
+    {
+        "identity" : "content-scope-scripts",
+        "kind" : "remoteSourceControl",
+        "location" : "https://github.com/duckduckgo/content-scope-scripts",
+        "state" : {
+            "revision" : "801b7f23476f797c6eaa72b070e6c80abb82801a",
+            "version" : "4.4.4"
+        }
+    },
+    {
+        "identity" : "duckduckgo-autofill",
+        "kind" : "remoteSourceControl",
+        "location" : "https://github.com/duckduckgo/duckduckgo-autofill.git",
+        "state" : {
+            "revision" : "4aee97d550112ba6551e61ea8019fb1f1a2d3af7",
+            "version" : "6.4.3"
+        }
+    },`
+
         const updatedPackageSwift = updatePackageSwift(examplePackageSwift, version)
         expect(updatedPackageSwift).toContain(`"https://github.com/duckduckgo/duckduckgo-autofill.git", .exact("${version}")`)
 
-        const updatedPackageResolved = updatePackageResolved(examplePackageResolved, version, commit)
-        expect(updatedPackageResolved).toContain(`"revision": "${commit}"`)
-        expect(updatedPackageResolved).toContain(`"version": "${version}"`)
+        const updatedPackageResolvedForBSK = updatePackageResolved(examplePackageResolvedForBSK, {autofill: {version, commit}})
+        expect(updatedPackageResolvedForBSK).toContain(`"revision" : "${commit}"`)
+        expect(updatedPackageResolvedForBSK).toContain(`"version" : "${version}"`)
+
+        const substitutions = {
+            autofill: {version, commit},
+            bsk: {commit: bskCommit}
+        }
+        const updatedPackageResolvedForMacOS = updatePackageResolved(examplePackageResolvedForMacOS, substitutions)
+        expect(updatedPackageResolvedForMacOS).toContain(`"revision" : "${commit}"`)
+        expect(updatedPackageResolvedForMacOS).toContain(`"version" : "${version}"`)
+        expect(updatedPackageResolvedForMacOS).toContain(`"revision" : "${bskCommit}"`)
```

Comment:
Is it work having a new test for this scenario? 
e.g. `test('BSK dependant can be updated successfully', ...`